exercises/knapsack/src/main/java/Knapsack.java

```diff
@@ -0,0 +1,29 @@
+import java.util.ArrayList;
```

Comment:
For this file, you may also use the [stub template](https://github.com/exercism/java/tree/master/_template/src/main/java), if it is an exercise with difficulty >4.

Reply:
Changing this file to use stub template, how would one know if the input is ```ArrayList``` or not?

Reply:
You will know it from the test cases.

Reply:
Done.